x509util/x509util.go

```diff
@@ -862,7 +862,7 @@ func ExtractSCT(sctData *x509.SerializedSCT) (*ct.SignedCertificateTimestamp, er
 }
 
 // MarshalSCTsIntoSCTList serializes SCTs into SCT list.
-func MarshalSCTsIntoSCTList(scts []*ct.SignedCertificateTimestamp) (*x509.SignedCertificateTimestampList, error) {
+func MarshalSCTsIntoSCTList(scts []*ct.SignedCertificateTimestamp) ([]byte, error) {
```

Comment:
Is that name still relevant now it no longer returns the x509 type of similar name?

Reply:
Moved the final encoding outside.